src/mongo/db/storage/error_count.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <cstdint>
+
+namespace mongo {
+struct ErrorCount {
+    // Variables to keep track of the count of errors occured during reading csv file.
+    std::int64_t _incompleteConversionToNumeric = 0;
+    std::int64_t _invalidInt32 = 0;
+    std::int64_t _invalidInt64 = 0;
+    std::int64_t _invalidDouble = 0;
+    std::int64_t _outOfRange = 0;
+    std::int64_t _invalidDate = 0;
+    std::int64_t _invalidOid = 0;
+    std::int64_t _invalidBoolean = 0;
+    std::int64_t _metadataAndDataDifferentLength = 0;
+    std::int64_t _totalErrorCount = 0;
+
+    ErrorCount operator+(const ErrorCount& other) const {
+        ErrorCount ret;
+        ret._incompleteConversionToNumeric =
+            this->_incompleteConversionToNumeric + other._incompleteConversionToNumeric;
+        ret._invalidInt32 = this->_invalidInt32 + other._invalidInt32;
+        ret._invalidInt64 = this->_invalidInt64 + other._invalidInt64;
+        ret._invalidDouble = this->_invalidDouble + other._invalidDouble;
+        ret._outOfRange = this->_outOfRange + other._outOfRange;
+        ret._invalidDate = this->_invalidDate + other._invalidDate;
+        ret._invalidBoolean = this->_invalidBoolean + other._invalidBoolean;
+        ret._invalidOid = this->_invalidOid + other._invalidOid;
+        ret._metadataAndDataDifferentLength =
+            this->_metadataAndDataDifferentLength + other._metadataAndDataDifferentLength;
+        ret._totalErrorCount = this->_totalErrorCount + other._totalErrorCount;
+        return ret;
```

Comment:
Can we reimplement this using += to remove repetitive code?

```suggestion
        ErrorCount ret(*this);
        ret += other;
        return ret;
```

Reply:
Done